app/constants/task-type.js

```diff
@@ -3,5 +3,7 @@ module.exports = {
   CREATE_WORKLOAD: 'CREATE-WORKLOAD',
   PROCESS_IMPORT: 'PROCESS-IMPORT',
   PROCESS_REDUCTIONS: 'PROCESS-REDUCTIONS',
-  PROCESS_ADJUSTMENTS: 'PROCESS-ADJUSTMENTS'
+  PROCESS_ADJUSTMENTS: 'PROCESS-ADJUSTMENTS',
+  CREATE_COURT_REPORTS: 'CREATE-COURT-REPORTS',
+  PROCESS_REDUCTIONS_COURT_REPORTERS: 'PROCESS-REDUCTIONS-COURT-REPORTER'
```

Comment:
`S` missing from the value - `REPORTERS`

Reply:
Added